src/styles/_variables.scss

```diff
@@ -1,7 +1,15 @@
 $light-grey: #AEAEAE;
 $light-cyan: #4DD0E1;
 $dark-blue: #0277BD;
+$photo-upload-panel: #5cb85c;
+$body-background: #FAFAFA;
 $font-arial: "Segoe UI", Helvetica, Arial, sans-serif;
 
 .light-grey { color: $light-grey };
 .change-photo, .user-nickname, .nickname { font-family: $font-arial};
+.bg-green { background-color: $photo-upload-panel !important; }
```

Comment:
use a double class to improve your index of the selector specificity and then u won't need the !important anymore